tfhe/src/core_crypto/commons/traits/contiguous_entity_container.rs

```diff
@@ -254,7 +254,7 @@ pub trait ContiguousEntityContainer: AsRef<[Self::Element]> {
     fn par_chunks<'this>(
         &'this self,
         chunk_size: usize,
-    ) -> ParallelChunksWrappingLendingIterator<'_, Self::Element, Self::SelfView<'_>>
+    ) -> ParallelChunksWrappingLendingIterator<'this, Self::Element, Self::SelfView<'this>>
```

Comment:
do you have the lint that this is triggered by ? It's weird the inference is not working here

Reply:
https://doc.rust-lang.org/nightly/nightly-rustc/rustc_lint/builtin/static.ELIDED_NAMED_LIFETIMES.html

Reply:
thanks that's very useful, so essentially the lifetime elision is not exactly "find the lifetime yourself"